ci/gh-actions.md

```diff
@@ -49,3 +49,29 @@ jobs:
     - name: Show current ref
       run: echo ${{ github.ref }}
 ```
+
+#### Stringify objects when using `core.*` utilities to output logs
+
+Objects are [expected to be supported](https://github.com/actions/toolkit/issues/386) sooner or later.
+
+GOOD
+
+```js
+try {
+  await run();
+} catch (err) {
+  core.setFailed(err.toString());
```

Comment:
Can you give an example of what kind of message `err.toString()` creates?

Reply:
Absolutely, good point, thanks!

cab684c